docs/source/guide/zne-2-use-case.myst

```diff
@@ -0,0 +1,43 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# When should I use ZNE?
+ 
+## Advantages 
+
+ZNE generally works well for circuits of low depth and/or those exhibiting noise 
+that can be described by a simple model 
+[[32](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)]. 
+The main advantage of ZNE is that the technique can be applied without knowledge of 
+the undelying noise model. Therefore it can be a good option in situations where 
+tomography is impractical.
+  
+
+## Disadvantages 
+
+In some instances the results of the extrapolation can exhibit a large bias
+[[33](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)].
+ZNE may not be helpful in cases where a low degree polynomial curve obtained
+by fitting the noisy expectation values does not match the zero-noise limit. 
+When using circuits of less trivial depth on real devices, the lowest error 
+points may be too noisy for the extrapolation to show improvement over the 
+unmitigated result
+[[32](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)]. 
+
+## Example
+
+For a simple example in which the application of ZNE reduces the estimation error when 
+compared to the unmitigated result, see:
+
+[Zero Noise Extrapolation for mitigating errors in energy landscape of a two-qubit 
+variational circuit](https://mitiq.readthedocs.io/en/latest/examples/simple_landscape.html)
```

Comment:
I like the idea of linking the example instead of copying the content.
I don't know what other people think, but I think this approach is optimal for all techniques.